add-on/src/landing-pages/welcome/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>IPFS Companion - Welcome</title>
```

Comment:
Are we able to add JS  that sets title to value from i18n ?

Reply:
I think we can leave this as is, but in the render of the `page.js` file we can override that to an i18n string! What do you think?

Reply:
Sounds good. 
We could also remove static `<title>` and just set it in the render to avoid any visual jitter in title bar on slow machines.